### PR TITLE
Feature/improve docker

### PR DESCRIPTION
**Pull request by sebbousquet**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+git checkout develop
+git pull origin develop
+git rebase origin/master
+
+npm --no-git-tag-version version "${DEV}-dev"
+
+git commit -a -m "Set development version to ${DEV}-dev"
+git push origin develop
+
+echo "==> Well done :)"
```
Comment (MohamedHamouGisaia) [inline release.sh:108]: Well done

File: src/app/components/left-menu/left-menu.component.ts [hunk 2]
```diff
@@ -115,7 +119,8 @@ export class LeftMenuComponent {
   }
 
   public save(event) {
-    if (this.persistenceService.isAvailable) {
+    if (this.persistenceService.isAvailable
+      || (this.persistenceService.isAvailable && this.persistenceService.isAuthenticate && this.persistenceService.isAuthAvailable)) {
```
Comment (MohamedHamouGisaia) [inline src/app/components/left-menu/left-menu.component.ts:123]: if `this.persistenceService.isAvailable` is true then the second part after || is true and same thing if it is false

File: src/app/services/main-form-manager/config-export-helper.ts [hunk 3]
```diff
@@ -256,9 +256,9 @@ export class ConfigExportHelper {
                         layerSource.include_fields.push(interpolatedValues.propertyInterpolatedFieldCtrl);
                     }
                 } else {
-                    if (!interpolatedValues.propertyInterpolatedCountOrMetricCtrl) {
+                    if (interpolatedValues.propertyInterpolatedCountOrMetricCtrl === 'count') {
                         layerSource.metrics.push({
-                            field: 'count',
+                            field: '',
```
Comment (MohamedHamouGisaia) [inline src/app/services/main-form-manager/config-export-helper.ts:261]: thanks

File: src/app/components/left-menu/left-menu.component.ts (hunk 2 above)
Comment (sebbousquet) [reply src/app/components/left-menu/left-menu.component.ts]: Fixed !
Too many conditions kill the condition

File: src/app/services/main-form-manager/config-export-helper.ts (hunk 3 above)
Comment (sebbousquet) [reply src/app/services/main-form-manager/config-export-helper.ts:261]: ;)

**Pull request merged**